### PR TITLE
Add HTS Precompile Acceptance Tests for Expiry Info methods

### LINKED ISSUE #408: Add htsPrecompile acceptance test support for auto renew methods
### Problem

No htsPrecompile acceptance test support for exists for auto renew methods

### Solution

Add support for token autoRenew

- `tokenGetAutoRenew(address)`
- `tokenUpdateAutoRenew(address, uint32)`

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by georgi-l95**
Signed-off-by: georgi-l95 <glazarov95@gmail.com>

**Description**:
Adds tests for:

- `getTokenExpiryInfo(address)` 
- `updateTokenExpiryInfo(address, Expiry)`

**Related issue(s)**:

Fixes #408 

**Notes for reviewer**:
Version of network node is bumped to 0.30.0-alpha.2 . Some adjustments before starting the node, so that acceptance tests can run. This unblocks all other PRs open for HTS Precompile tests.

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: packages/server/tests/acceptance/htsPrecompile.spec.ts [hunk 1]
```diff
@@ -586,6 +586,65 @@ describe('HTS Precompile Acceptance Tests', async function () {
...
 
+  // Depends on https://github.com/hashgraph/hedera-services/pull/3756
+  xdescribe('HTS Precompile Token Expiry Info Tests', async function() {
+    it('should be able to get fungible token expiry info', async function() {
+      const getTokenExpiryInfoTx = (await baseHTSContract.getTokenExpiryInfoPublic(HTSTokenContractAddress));
+      const responseCode = (await getTokenExpiryInfoTx.wait()).events.filter(e => e.event === 'ResponseCode')[0].args.responseCode;
+      const tokenExpiryInfo = (await getTokenExpiryInfoTx.wait()).events.filter(e => e.event === 'TokenExpiryInfo')[0].args.expiryInfo;
+
+      expect(responseCode).to.equal(TX_SUCCESS_CODE);
+      expect(tokenExpiryInfo.autoRenewPeriod).to.equal(7776000);
```
Comment (Nana-EC) [inline packages/server/tests/acceptance/htsPrecompile.spec.ts:597]: nit: add check for `second` also here and below

File: packages/server/tests/acceptance/htsPrecompile.spec.ts [hunk 2]
```diff
@@ -586,6 +586,65 @@ describe('HTS Precompile Acceptance Tests', async function () {
...
+      const getTokenExpiryInfoTx = (await baseHTSContract.getTokenExpiryInfoPublic(NftHTSTokenContractAddress));
+      const responseCode = (await getTokenExpiryInfoTx.wait()).events.filter(e => e.event === 'ResponseCode')[0].args.responseCode;
+      const tokenExpiryInfo = (await getTokenExpiryInfoTx.wait()).events.filter(e => e.event === 'TokenExpiryInfo')[0].args.expiryInfo;
+
+      expect(responseCode).to.equal(TX_SUCCESS_CODE);
+      expect(tokenExpiryInfo.autoRenewPeriod).to.equal(7776000);
+      expect(tokenExpiryInfo.autoRenewAccount).to.equal(NftHTSTokenContractAddress);
+    });
+
+    it('should be able to update fungible token expiry info', async function() {
```
Comment (Nana-EC) [inline packages/server/tests/acceptance/htsPrecompile.spec.ts:611]: Improvement suggestion.
Change these 4 tests into 2 tests

Test 1 ( combines your current test 1 and 3)
- confirm `getTokenExpiryInfoPublic()` for FT
- update expiry info using `updateTokenExpiryInfoPublic()`
- confirm updated info


Test 2 ( combines your current test 2 and 4)
- confirm `getTokenExpiryInfoPublic()` for NFT
- update expiry info using `updateTokenExpiryInfoPublic()`
- confirm updated info

**Review by Nana-EC — CHANGES_REQUESTED** [review]
LG.
Layout suggestion.
Once addressed please move this to a draft PR.
I'd prefer to not check in turned of tests so we don't forget but rather just hold off till we get the 0.30.0 services tag next sprint.

File: packages/server/tests/acceptance/htsPrecompile.spec.ts (hunk 2 above)
Comment (georgi-l95) [reply packages/server/tests/acceptance/htsPrecompile.spec.ts]: Done. Updated the flow.

**Comment by georgi-l95** [comment]
Moving this PR to draft, until we get the 0.30.0 services tag next sprint.

File: packages/server/tests/acceptance/htsPrecompile.spec.ts (hunk 1 above)
Comment (georgi-l95) [reply packages/server/tests/acceptance/htsPrecompile.spec.ts]: Done

**Review by Nana-EC — APPROVED** [review]
LG
Please open a ticket to track us coming back and removing this workaround.
Thanks

**Pull request merged**